packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -440,6 +440,7 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection {
     }
 
     protected serverDisconnectHandler = (error: IFluidErrorBase & OdspError) => {
+        this.logger.sendTelemetryEvent({eventName: "ServerDisconnectOdsp"}, error);
```

Comment:
nit - logger already says "OdspDriver" in namespace so could probably drop "Odsp" from the name.  Your call though, it's fine as-is too.